scripts/gdb/gdbrc

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+#
+# gdb startup script for "wrapper" execution.
+#
+# Use this as gdb startup script, for example:
+#
+#     export GDBPARAMS="-x $PWD/scripts/gdb/gdbrc"; ./utils/m0run -dgdb m0ut -- ...
+#
+# The script runs the executable. If the executable completes normally (that is,
+# by calling exit(2)) the scripts exits the gdb. Otherwise (i.e., when the
+# process terminates anbormally), gdb session remains active.
```

Comment:
s/anbormally/abnormally/

Reply:
@nikitadanilov : Can you fix above spelling error ?